jabgui/src/main/java/org/jabref/gui/theme/ThemeManager.java

```diff
@@ -83,6 +96,58 @@ public ThemeManager(WorkspacePreferences workspacePreferences,
         updateThemeSettings();
     }
 
+    private void initializeWindowThemeUpdater(boolean darkMode) {
+        if (!SUPPORTS_DARK_MODE) {
+            return;
+        }
+
+        this.isDarkMode = darkMode;
+
+        ListChangeListener<Window> windowsListener = change -> {
+            while (change.next()) {
+                if (!change.wasAdded()) {
+                    continue;
+                }
+                change.getAddedSubList().stream()
+                      .filter(Stage.class::isInstance)
+                      .map(Stage.class::cast)
+                      .forEach(stage -> {
+                          BindingsHelper.subscribeFuture(stage.showingProperty(), showing -> {
+                              if (showing) {
+                                  applyDarkModeToWindow(stage, isDarkMode);
+                              }
+                          });
+                          if (stage.isShowing()) {
+                              applyDarkModeToWindow(stage, isDarkMode);
+                          }
```

Comment:
duplication can be solved with EasyBind::subscribe

Reply:
i'm not sure but i think this may cause a memory leak. if a window is closed, the reference is not removed and the garbage collector wont remove this. or am i wrong here? @Siedlerchr @koppor

Reply:
@Yubo-Cao do not close converstations with questions raised that have not been answered. Maybe you already know, but I want to know.

Reply:
For resolving just write a "done" or something like this if you addressed the comment